src/main/scala/redis/commands/Transactions.scala

```diff
@@ -112,12 +112,10 @@ case class Transaction(watcher: Set[String], operations: Queue[Operation[_, _]],
 
   def dispatchExecReply(multiBulk: MultiBulk) = {
     multiBulk.responses.map(replies => {
-      (replies, operations).zipped.map((reply, operation) => {
-        reply match {
-          case e: Error => operation.completeFailed(ReplyErrorException(e.toString()))
-          case _ => operation.tryCompleteSuccess(reply)
-        }
-      })
+      replies.zip(operations).map { 
```

Comment:
The 2.13 compiler recommends using `.lazyZip` here but that's not backward compatible and I think we don't need lazy evaluation here.